# Ejercicio 2

 
En un Ecommerce en el que se venden 3 productos:

Libretas 100 hojas
* Venta al público $20 pesos
* Utilidad bruta $5 pesos
* Utilidad bruta total 2023 $60,000

Lápices $5 pesos
* Venta al público $20 pesos
* Utilidad bruta $1.5 pesos
* Utilidad bruta total 2023 $90,000

Agendas $40 pesos
* Venta al público $20 pesos
* Utilidad bruta $5 pesos
* Utilidad bruta total 2023 $24,000
 

En 2024 se planea el lanzamiento del producto “libretas 50 hojas” con un costo de $12 pesos (se estima una utilidad bruta por pieza $3). La hipótesis del departamento de nuevos productos es; “las libretas de 100 hojas se dejarán de vender ya que los clientes preferirán gastar menos y comprarán las de 50 hojas”. Si la premisa fuera real necesitamos saber cuál es el impacto a negocio resolviendo los siguientes puntos:

 

Para el año 2024 (considerar el año completo)
* ¿Cuál será la utilidad bruta de las libretas de 100 hojas si las ventas se contraen en 40% respecto 2023?

* Si las ventas de las libretas de 50 hojas cubren el volumen perdido de las de 100 hojas ¿de cuánto será la utilidad bruta de las libretas de 50 hojas en 2024?

* ¿Cuál será la utilidad bruta total compañía en 2024 si las ventas de los lápices y las agendas crecen 30% respecto 2023?

* De acuerdo con los resultados anteriores ¿será eficiente para el negocio llevar a cabo el lanzamiento de las libretas de 50 hojas si el porcentaje de crecimiento esperado a total compañía para 2024 es de 25%?

* ¿Cuál sería tu recomendación al negocio?

## ¿Cuál será la utilidad bruta de las libretas de 100 hojas si las ventas se contraen en 40% respecto 2023?

r = 36,000

In [23]:
public_sale = 20 
gross_profit_per_unit = 5 
total_gross_profit_2023 = 60000 
contraction_percentage = 40 


units_sold_2023 = total_gross_profit_2023 / gross_profit_per_unit 

units_sold_2024_with_contraction = units_sold_2023 * (1 - contraction_percentage / 100) 

gross_profit_2024_correct = units_sold_2024_with_contraction * gross_profit_per_unit

gross_profit_2024_correct

36000.0

## Si las ventas de las libretas de 50 hojas cubren el volumen perdido de las de 100 hojas ¿de cuánto será la utilidad bruta de las libretas de 50 hojas en 2024?

r = 14,400

In [18]:
utility_per_unit_50_sheets = 3

unit_volume = units_sold_2023 - units_sold_2024_with_contraction
utility_notebook_50_2024 = unit_volume * utility_per_unit_50_sheets

utility_notebook_50_2024


14400.0

## ¿Cuál será la utilidad bruta total de la compañía en 2024 si las ventas de los lápices y las agendas crecen 30% respecto 2023?

r = 198600

In [20]:
gross_utility_total_pencils_2023 = 90000
gross_profit_total_agendas_2023 = 24000
gross_profit_total_notebooks_100_2023 = gross_profit_2024_correct
gross_profit_total_notebooks_50_2023 = utility_notebook_50_2024

percentage = 0.30

gross_utility_pencils_2024 = gross_utility_total_pencils_2023 * (1 + percentage)
gross_profit_agendas_2024 = gross_profit_total_agendas_2023 * (1 + percentage)

gross_profit_total_company_2024 = ( gross_profit_total_notebooks_100_2023 + 
                                    gross_utility_pencils_2024 + 
                                    gross_profit_total_notebooks_50_2023 +
                                    gross_profit_agendas_2024)

gross_profit_total_company_2024


198600.0

## De acuerdo con los resultados anteriores ¿será eficiente para el negocio llevar a cabo el lanzamiento de las libretas de 50 hojas si el porcentaje de crecimiento esperado a total compañía para 2024 es de 25%?

r = Basado en los cálculos realizados, si la empresa establece como objetivo un crecimiento del 25% en su utilidad bruta total para 2024, y los resultados obtenidos indican solo un crecimiento del 14.1379%, lanzar las libretas de 50 hojas bajo las condiciones actuales no parece ser la estrategia más eficiente para alcanzar sus metas financieras. Este análisis sugiere que, aunque la introducción de las libretas de 50 hojas puede compensar parcialmente la contracción en las ventas de las libretas de 100 hojas, <strong>no contribuye suficientemente al crecimiento general de la empresa como se había previsto.</strong>

In [22]:
gross_profit_total_company_2023 = total_gross_profit_2023 + gross_utility_total_pencils_2023 + gross_profit_total_agendas_2023

growth_percentage_2024 = ((gross_profit_total_company_2024 - gross_profit_total_company_2023) / gross_profit_total_company_2023) * 100

growth_percentage_2024


14.13793103448276

## Recomendación al Negocio

Dado que el crecimiento proyectado de la utilidad bruta total de la compañía con el lanzamiento de las libretas de 50 hojas es del 14.14% aprox, lo cual no cumple con el objetivo de crecimiento del 25%, recomendaría revisar la estrategia para el lanzamiento de este nuevo producto. Buscando alternativas para aumentar la utilidad bruta de los productos actuales, o bien, replantear la estrategia de lanzamiento de las libretas de 50 hojas, con el fin de que contribuya de manera más significativa al crecimiento de la empresa.